src/index.js

```diff
@@ -43,6 +44,9 @@ function tokenIntrospect(opts = {}) {
     try {
       return await localIntrospect(token, tokenTypeHint);
     } catch (err) {
+      if (err instanceof jwt.TokenExpiredError || err instanceof jwt.NotBeforeError) {
+        throw err;
+      }
       debug('Could not verify token: %s', err.message);
```

Comment:
Should this line be moved before the re-throwing of exception to make sure these cases are also logged?